app/views/pages/_topics.html.erb

```diff
@@ -1,69 +1,64 @@
-<div class="grid-row dgu-topics">
-  <div class="column-one-third">
-    <div>
-      <ol>
-        <li>
-          <h2><%= link_to t('.business_and_economy_label'), search_path(filters: { topic: 'Business and economy' }), class: 'govuk-link' %></h2>
-          <p><%= t('.business_and_economy_description') %></p>
-        </li>
-        <li>
-          <h2><%= link_to t('.crime_and_justice_label'), search_path(filters: { topic: 'Crime and justice' }), class: 'govuk-link' %></h2>
-          <p><%= t('.crime_and_justice_description') %></p>
-        </li>
-        <li>
-          <h2><%= link_to t('.defence_label'), search_path(filters: { topic: 'Defence' }), class: 'govuk-link' %></h2>
-          <p><%= t('.defence_description') %></p>
-        </li>
-        <li>
-          <h2><%= link_to t('.education_label'), search_path(filters: { topic: 'Education' }), class: 'govuk-link' %></h2>
-          <p><%= t('.education_description') %></p>
-        </li>
-      </ol>
-    </div>
+<div class="govuk-grid-row">
+  <h2 class="govuk-visually-hidden"><%= t('.topics_heading') %></h2>
```

Comment:
Larger comment on the three columns list - the way this is coded means that there are three lists disguised as one list.

This means that people who use screen readers (and similar assistive technologies) will have to navigate three lists - but for people who use the site visually is looks like one list.

Ideally this would be one list with twelve items, rather than three lists with four items each.

I don't know what DGU's browser support is, but assuming it's the same as GOV.UK Publishing we could look at doing this with [CSS columns](https://caniuse.com/#search=columns) with a single column fallback.